apache/drill
# DRILL-6777: Successful implementation of CircleCI build for Drill

- adding .circleci/config.yml to the project to launch CircleCI
- custom memory parameters
- usage of CircleCI machine
- excluding "SlowTest" and "UnlikelyTest" groups
- update maven version
- adding libaio.so library to solve MySQL integration tests
- update com.jcabi:jcabi-mysql-maven-plugin library version
- TODO descriptions for the future enhancements of CircleCI build for Drill

see details in [DRILL-6777](https://issues.apache.org/jira/browse/DRILL-6741) and [DRILL-6741](https://issues.apache.org/jira/browse/DRILL-6777)
